stan/math/prim/functor/map_rect_mpi.hpp

```diff
@@ -1,14 +1,14 @@
 #ifdef STAN_MPI
 
-#ifndef STAN_MATH_PRIM_MAT_FUNCTOR_MAP_RECT_MPI_HPP
+#ifndef STAN_MATH_PRIM_FUNCTOR_MAP_RECT_MPI_HPP
 #include <stan/math/prim/meta.hpp>
-#define STAN_MATH_PRIM_MAT_FUNCTOR_MAP_RECT_MPI_HPP
+#define STAN_MATH_PRIM_FUNCTOR_MAP_RECT_MPI_HPP
```

Comment:
Preexisting: does this define need to be after including meta? I've never seen this done anywhere else, so it looks like it could be tidied up.

Reply:
Yeah, this need to be cleaned up. I missed it.